### ISSUE TITLE
Backtrace for newtons_crade.jl

### ISSUE THREAD

**Issue by sje30** [jheinen/GR.jl#9]
Hi,
I get the following backtrace running newtons_cradle (a window appears with a horizontal box  and a vertical line).  This is with julia HEAD and presumably the latest version of GR.

I've got this on both my mac and linux.  Let me know if I can help further.  (I have edited the warnings about int(x))

```
julia> include("newtons_cradle.jl")
WARNING: int(x) is deprecated, use Int(x) instead.
 in depwarn at ./deprecated.jl:73
...
while loading /mhome/damtp/cbi/sje30/.julia/v0.4/GR/examples/newtons_cradle.jl, in expression starting on line 46
WARNING: int(x) is deprecated, use Int(x) instead.
 in depwarn at ./deprecated.jl:73
 while loading /mhome/damtp/cbi/sje30/.julia/v0.4/GR/examples/newtons_cradle.jl, in expression starting on line 46
ERROR: LoadError: InexactError()
 in copy! at abstractarray.jl:307
 in drawimage at /home/sje30/.julia/v0.4/GR/src/GR.jl:809
 in draw_cradle at /mhome/damtp/cbi/sje30/.julia/v0.4/GR/examples/newtons_cradle.jl:33
 [inlined code] from /mhome/damtp/cbi/sje30/.julia/v0.4/GR/examples/newtons_cradle.jl:55
 in anonymous at no file:0
 in include at ./boot.jl:260
 in include_from_node1 at ./loading.jl:271
 in eval_user_input at REPL.jl:64
 [inlined code] from REPL.jl:93
 in anonymous at task.jl:68
while loading /mhome/damtp/cbi/sje30/.julia/v0.4/GR/examples/newtons_cradle.jl, in expression starting on line 50
```

**Comment by jheinen** [comment]
I've fixed this v0.4 incompatibility. The Julia examples should now run without any problems both in v0.3 and v0.4.

**Comment by sje30** [comment]
Thank you, it is now working for me.

And wow, very nice demo!!!
